hbw/app/javascript/packs/hbw/components/form/string.js.jsx

```diff
@@ -87,9 +87,14 @@ modulejs.define('HBWFormString', ['React'], (React) => {
     }
 
     componentDidMount () {
+      const { task } = this.props;
+      const tooltipMessage = this.context.translateBP(`${task.process_key}.${task.key}.${this.props.name}.message`,
+        {},
+        this.props.params.message || this.context.translate('errors.field_is_required'));
```

Comment:
prefetch `name` and `params` from props at line 90 please, keep the style consistent